src/docs/stan-reference/appendices.tex

```diff
@@ -724,7 +745,7 @@ \subsection{Statements}
 
 truncation ::= 'T' '[' ?expression ',' ?expression ']'
 
-lhs ::= identifier ?('[' indexes ']')
+lhs ::= identifier ('[' indexes ']')*
```

Comment:
I didn't even realize the language supported `a[2][3] = 5;`, but I just verififed and the new version's correct.  Thanks for updating.

Reply:
Neither did I.  But it's somewhere in all those tests.